README.md

```diff
@@ -72,7 +95,11 @@ Anyone and everyone is welcome to [contribute](https://github.com/Esri/maps-app-
 3. Contribute code
 4. Improve documentation
 
-### Licensing
+## MDTOC
+
+Generation of this and other document's table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```

Comment:
I realized I got this wrong in one of our initial PRs and we've been carrying through the change since. Possessive apostrophe should come at the end since we're referring to the plural form of "documents" -

```suggestion
Generation of this and other documents' table of contents in this repository was performed using the [MDTOC package for Atom](https://atom.io/packages/atom-mdtoc).
```